.pipeline/lib/build.js

```diff
@@ -11,10 +11,12 @@ module.exports = (settings)=>{
   const templatesLocalBaseUrl =oc.toFileUrl(path.resolve(__dirname, '../../openshift'))
 
   // The building of your cool app goes here ▼▼▼
-    objects = objects.concat(oc.processBuildTemplate(`${templatesLocalBaseUrl}/bc.yaml`, {
+    objects = objects.concat(oc.processDeploymentTemplate(`${templatesLocalBaseUrl}/bc.yaml`, {
```

Comment:
I am just curious  why we want to use __processDeploymentTemplate__ for 'bc.yaml'

Reply:
i'm not sure!